### PR TITLE
Milestone 04: Add UI :pause_button: 

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### In this branch, I (@HAMMAS-SALEEM ) have worked on the following things :+1: 

:heavy_check_mark: Build School Library app interactive
:heavy_check_mark: Followed Ruby best practices
:heavy_check_mark: Followed Linters
:heavy_check_mark: Followed Gitflow
:heavy_check_mark: Documented work in professional way

### CODE REVIEW

File: classes/app.rb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      specialization = gets.chomp
+      teacher = Teacher.new(specialization, age, name)
+      @people.push(teacher)
+    end
+  end
+
+  def list_people
+    puts 'No person available' if @people.length.zero?
+    @people.each { |person| puts "Name: #{person.name} Age: #{person.age}" }
+    back_to_menu
```
Comment (Gambit142) [inline classes/app.rb]: - [ ] Great work done with your console app. Nice 👏. There is just one more significant feature for you to implement. If you watch the video, when you input `2`, it should show all the people created with the type of class, i.e either a `Student` or `Teacher`. The output should look like this:

image

Kindly implement this feature. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @HAMMAS-SALEEM 👋, 

Good job so far!

### Highlights

- Implemented UI design based on business requirements seen in the video 💯 
- Linters checks passed 💯 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: classes/app.rb (hunk 1 above)
Comment (HAMMAS-SALEEM) [reply classes/app.rb]: Thanks i would implement this feature

Comment (Gambit142) [reply classes/app.rb]: I am glad you are going to implement it.

Comment (HAMMAS-SALEEM) [reply classes/app.rb]: I have implemented this feature :+1:

**Review by rukundoeric — APPROVED** [review]
Hi @HAMMAS-SALEEM,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**